x-pack/plugins/maps/public/components/geo_index_pattern_select.tsx

```diff
@@ -57,24 +59,31 @@ export class GeoIndexPatternSelect extends Component<Props, State> {
     // method may be called again before 'get' returns
     // ignore response when fetched index pattern does not match active index pattern
     if (this._isMounted && indexPattern.id === indexPatternId) {
+      this.setState({
+        doesIndexPatternHaveGeoField: indexPattern.fields.some((field) => {
+          return this.props?.isGeoPointsOnly
+            ? [ES_GEO_FIELD_TYPE.GEO_POINT as string].includes(field.type)
```

Comment:
We could just say `field.type === ES_GEO_FIELD_TYPE.GEO_POINT` rather than wrap it in an array